modules/csharp/examples/code_snippets/VectorSearch.cs

```diff
@@ -0,0 +1,203 @@
+﻿//
+// VectorSearch.cs
+//
+// Copyright (c) 2024 Couchbase, Inc All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using Couchbase.Lite;
+using Couchbase.Lite.Enterprise.Query;
+using Couchbase.Lite.Extensions;
+using Couchbase.Lite.Query;
+
+namespace api_walkthrough
+{
+    class Color
+    {
+        public static float[] GetVector(string color)
+        {
+            return new float[] { 0.0f, 0.0f, 0.0f };
+        }
+
+        public string Name { get; set; }
+    }
+
+    public class VectorSearch
+    {
+        public VectorSearch() { }
+
+        private static void EnableVectorSearchExtension()
+        {
+            // tag::vs - setup - packaging[]
+            Extension.Load(new VectorSearchExtension());
+            // end::vs - setup - packaging[]
+        }
+
+        private void CreateDefaultVectorIndexConfig()
+        {
+            // tag::vs - create - default - config[]
+            // Create a vector index configuration for indexing 3 dimensional vectors embedded
+            // in the documents' key named "vector" using 2 centroids.
+            var config = new VectorIndexConfiguration("vector", 3, 2);
+            // end::vs - create - default - config[]
+        }
+
+        private void CreateCustomVectorIndexConfig()
+        {
+            // tag::vs - create - default - config[]
```

Comment:
presumably this should be `tag::vs-create-custom-config[]` e.g. default -> custom